turbo.json

```diff
@@ -17,6 +17,23 @@
     },
     "clean": {
       "cache": false
+    },
+    "@latticexyz/store#clean": {
+      "dependsOn": [
+        "@latticexyz/world-module-erc20#clean",
+        "@latticexyz/world-modules#clean",
+        "puppet-modules-test#clean",
+        "mock-game-contracts#clean"
+      ],
+      "cache": false
+    },
+    "@latticexyz/world#clean": {
+      "dependsOn": [
+        "@latticexyz/world-module-metadata#clean",
+        "@latticexyz/world-module-erc20#clean",
+        "@latticexyz/world-modules#clean"
+      ],
+      "cache": false
```

Comment:
just so I understand, this is like an "inverse dependency"? cleaning the stuff that depends on store/world first before cleaning store/world

Reply:
yes exactly